src/printapp/printstatus.py

```diff
@@ -166,6 +170,7 @@ def fetch_data(self):
         for i in range(numberOfJobs):
             j = i * 7
```

Comment:
Sort of. `j` is the index of all the `<td>` tags. So we have an array of all the `<td>` tags of the table the queue is in. Each row of the table has 7 columns. So `j` serves as a base index for each row.

I could just write
```
name = unicode(print_job_tags[0 + i * 7].string)
```
but I think that's more confusing.